http/lib.sh

```diff
@@ -665,6 +671,25 @@ httpRestoreMod() {
     return $ret
 }
 
+httpSetMPM() {
+    if [[ $# = 0 ]]; then
+        rlLogError "No Multi-Processing Module specified"
+        return 1
+    fi
+    ret=0
+    module=$1
+    mpmconf=${httpCONFDIR}/conf.modules.d/00-mpm.conf
+
+    # disable all modules
+    sed -i 's/^LoadModule/#LoadModule/' $mpmconf || ret=1
+    # enable chosen one
+    sed -i "s/#LoadModule mpm_${module}/LoadModule mpm_${module}/" $mpmconf || ret=1
+
+    [[ $ret -eq 0 ]] && rlLogInfo "MPM module has been set to $module."
```

Comment:
You use inconsistent comparison operators but I can merge it if you insist (FYI '-eq' is preferable over '=').

Reply:
(inconsistent with line 675)

Reply:
Thanks, fix pushed.